Framework/Core/src/CommonServices.cxx

```diff
@@ -1105,6 +1108,9 @@ o2::framework::ServiceSpec CommonServices::dataProcessingStats()
                    .sendInitialValue = true}};
 
       for (auto& metric : metrics) {
+        if (metric.metricId == (int)ProcessingStatsId::AVAILABLE_MANAGED_SHM_BASE + (runningWorkflow.shmSegmentId % 512) && spec.name.compare("readout-proxy") != 0) {
```

Comment:
Is "readout-proxy" the only name we have?

Reply:
it is the only name we use in the online processing, and we only display it for this task in Grafana. The calibration workflows have different input proxy names each, and (at least so far) we don't look at their shm.